eipw-lint/src/lints/markdown/relative_links.rs

```diff
@@ -38,13 +42,34 @@ impl<'e> Lint for RelativeLinks<'e> {
             .filter(|l| re.is_match(&l.address) && !exceptions.is_match(&l.address));
 
         for Link { line_start, .. } in links {
+            
+            let mut link_md = String::new();
+            let line_with_address = ctx.line(line_start);
+            let line_link_regex = re.captures(line_with_address.as_bytes()).unwrap();
+            let line_link_address = line_link_regex.get(0).unwrap();
+            
+            write!(link_md, "`{:?}`",&line_link_address).unwrap();
```

Comment:
The `{:?}` placeholder outputs the debug format of the given argument. That's why you're getting error messages like:

```
error[markdown-rel]: non-relative link or image
  |
5 | [hi](https://example.com/)
  |
  = help: use "`Match { text: [91, 104, 105, 93, 40, 104, 116, 116, 112, 115, 58, 47, 47, 101, 120, 97, 109, 112, 108, 101, 46, 99, 111, 109, 47, 41], start: 10, end: 13 }`" instead
```